app/views/authors/about.html.erb

```diff
@@ -2,10 +2,10 @@
   </div>
   <div class="author-info">
     <h3 class="author-title">
-        <%= Author.find("Brown").show_name %>
+        <%= Author.find_by_last_name("Brown").show_name %>
```

Comment:
Try to avoid methods that hit the database in views. Either set @author in the controller or use @post.author/similar.

Remember: Keep your views _dumb_